functions/nodejs-apps.sh

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
 
 nodejs_setup() {
-  if ! command -v npm &>/dev/null; then
+  if command -v npm &>/dev/null; then
+    return 0
+  else
```

Comment:
the else block isn't needed. just end with `fi`. It's a small change but will make everything a bit clearer

Reply:
Nevermind about this minor detail. I'll merge now